examples/fluids/problems/freestream_bc.c

```diff
@@ -11,20 +11,20 @@
 #include "../qfunctions/freestream_bc.h"
 
```

Comment:
I tried removing this random extra line, but clang-format seems to want it for some reason. Any ideas why?

Reply:
I think it's because the `.h` filename matches this `.c` source file. If you include a header like that later, it can hide bugs (where the header doesn't include things it uses, and thus can break if it's included from a different source file). So it's a good rule.